### PR TITLE
Job Explorer Feature

### PR DESCRIPTION
**Pull request by kialam**
Related #127

### CODE REVIEW

File: src/Components/Toolbar.js [hunk 1]
```diff
@@ -0,0 +1,460 @@
...
+            collapseListedFiltersBreakpoint="xl"
+        >
+            <ToolbarContent>
+                <ToolbarToggleGroup toggleIcon={ <FilterIcon /> } breakpoint="xl">
+                    { dateRanges.length > 0 && (
+                        <ToolbarGroup variant="filter-group">
+                            { buildCategoryDropdown(toolbarCategories) }
+                            { buildFilterDropdown() }
+                            { passedFilters.date === 'custom' && (
+                <>
```
Comment (jctanner) [inline src/Components/Toolbar.js:383]: Missing slash?

Comment (kialam) [reply src/Components/Toolbar.js:383]: `<>` and `</>` is the shorthand for `React.Fragment`.

**Comment by brumik** [comment]
@kialam This branch is not compiling with `npm start` because of linter errors in the `tower-analytics-frontend/src/Icons/StatusIcon/StatusIcon.js` Nothing that automatically cannot be fixed.

File: src/Charts/LineChart.js [hunk 2]
```diff
@@ -11,12 +18,29 @@ class LineChart extends Component {
...
+            end_date: formattedDate,
+            quick_date_range: 'custom'
+        };
+        const { strings, stringify } = formatQueryStrings(initialQueryParams);
+        const search = stringify(strings);
+        this.props.history.push({
+            pathname: jobExplorer,
+            search
+        });
+    };
```
[the remark this answers is not in the document]
Comment (kialam) [reply src/Charts/LineChart.js:42]: It looks like as we continue to flesh out this feature and integrate it with the rest of our views we will probably need some reusable hook of some sort to redirect users from various views to the job explorer. I will break this out as a separate task after we merge this.

**Pull request merged**